keyboards/durgod/k320/k320.h

```diff
@@ -42,3 +42,22 @@ void on_all_leds(void);
     { K50,  K51,  K52,  XXX,  XXX,  XXX,  K56,  XXX,  XXX,  XXX,  K5A,  K5B,  K5C,  K5D,  K5E,  K5F },  \
     { XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  XXX,  K6F }   \
 }
+
+// This a shortcut to help you visually see your layout.
+#define LAYOUT_tkl_iso( \
+      K00,  K01,  K02,  K03,  K04,  K05,  K06,  K07,  K08,  K09,  K0A,  K0B,  K0C,  K0D,  K0E,  K0F,  \
+      K10,  K11,  K12,  K13,  K14,  K15,  K16,  K17,  K18,  K19,  K1A,  K1B,  K1C,        K1E,  K1F,  \
+      K20,  K21,  K22,  K23,  K24,  K25,  K26,  K27,  K28,  K29,  K2A,  K2B,  K2C,        K2E,  K2F,  \
+      K30,  K31,  K32,  K33,  K34,  K35,  K36,  K37,  K38,  K39,  K3A,  K3B,  K3C,  K3D,  K3E,  K3F,  \
+      K40,  K41,  K42,  K43,  K44,  K45,  K46,  K47,  K48,  K49,  K4A,  K4B,        K4D,  K4E,  K4F,  \
+      K50,  K51,  K52,                    K56,                    K5A,  K5B,  K5C,  K5D,  K5E,  K5F,  \
+                                                                                                K6F   \
```

Comment:
This's nothing technically wrong with this, but you might want to re-arrange the way the keys are listed here so that it matches the actual keyboard layout. As it it now, it is somewhat confusing...

Reply:
Thanks - done, I think... PTAL.

I've made the changes for the two ISO layouts in this PR, I'm a little reluctant to try to fix up the ANSI version as I don't have an ANSI version of the keyboard.